utils/copyFile.js

```diff
@@ -0,0 +1,5 @@
+const fs = require('fs/promises');
+
+const copyFile = (path, destination) => fs.copyFile(path, destination);
+
+module.exports = copyFile
```

Comment:
remote this util and use native  fs.copyFile(path, destination), beciuse is redundantly